build-tools/create-packs/SignList.xml

```diff
@@ -26,5 +26,12 @@
     <FirstParty Include="*.exe" />
     <!-- First and third party .jar files use the same Authenticode -->
     <FirstParty Include="*.jar" />
+    <MacDeveloperSign Include="*.dylib" />
+    <MacDeveloperSignHarden Include="Darwin/aapt2" />
+    <MacDeveloperSignHarden Include="Darwin/binutils/bin/*as" />
+    <MacDeveloperSignHarden Include="Darwin/binutils/bin/*ld" />
+    <MacDeveloperSignHarden Include="Darwin/binutils/bin/llc" />
+    <MacDeveloperSignHarden Include="Darwin/binutils/bin/llvm-mc" />
+    <MacDeveloperSignHarden Include="Darwin/binutils/bin/*strip" />
```

Comment:
Previously, we used an item group like `@(_MSBuildFilesUnixSignAndHarden)` or `@(_BinUtilsFilesUnixSign)`, is there a chance we might miss a file?